tests/Io/FileSystemTest.php

```diff
@@ -42,6 +42,132 @@ public function tearDown()
         $this->exec('rm -rf ' . $this->tmp);
     }
 
+    /**
+     * @expectedException        Brick\Std\Io\IoException
+     * @expectedExceptionMessage Error getting real path of invalid_path, check that the path exists
+     */
+    public function testGetRealPathWithInvalidPath()
+    {
+        FileSystem::getRealPath('invalid_path');
+    }
+
+    public function testGetRealPath()
+    {
+        $filePath = __DIR__ . '/../../composer.json';
+        $expectedRealPath = realpath($filePath);
+
+        $this->assertSame($expectedRealPath, FileSystem::getRealPath($filePath));
+    }
+
+    public function testWriteWithAppendFlag()
+    {
+        FileSystem::write('./temp_file', 'data1' . PHP_EOL);
+        FileSystem::write('./temp_file', 'data2', true);
+
+        $this->assertSame('data1' . PHP_EOL . 'data2', FileSystem::read('./temp_file'));
+    }
+
+    public function testWriteWithLockFlag()
+    {
+        $this->assertSame(0, FileSystem::write('./temp_lock_file', '', false, true));
```

Comment:
Can you `write()` a few chars in here, and test that the return value matches the length?